services/src/main/java/edu/unc/lib/dl/cdr/services/rest/modify/MoveObjectsController.java

```diff
@@ -59,6 +59,7 @@ public MoveObjectsController() {
         if (moveRequest == null || moveRequest.moved == null || moveRequest.moved.size() == 0
                 || moveRequest.getDestination() == null || moveRequest.getDestination().length() == 0) {
             results.put("error", "Request must provide a destination destination and a list of ids");
+            log.error("Request must provide a destination destination and a list of ids");
```

Comment:
While we're here, we might as well deduplicate the "destination" text here and on line 61

Reply:
aw, double destination is twice as nice :-)